### PR TITLE
feat: update bindings

### PR DESCRIPTION
**Pull request by azurwastaken**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Feature

## What is the current behavior?

Resolves: #55 

## What is the new behavior?

Handle new version of vesu's liquidate contract

## Does this introduce a breaking change?

No

### CODE REVIEW

File: src/services/monitoring.rs [hunk 1]
```diff
@@ -142,8 +142,12 @@ impl MonitoringService {
             .fetch_liquidation_factors(&self.config, self.rpc_client.clone())
             .await;
 
-        let debt_to_liquidate =
-            liquidable_amount_as_debt_asset.clone() * liquidation_factor.clone();
+        let debt_to_liquidate = match self.config.liquidation_mode {
```
Comment (akhercha) [inline src/services/monitoring.rs:145]: As we said, this need a little more in depth refactoring. Currently the code is shared but in a Full mode we don't need all those computation.

**Review by akhercha — CHANGES_REQUESTED** [review]
One smol comment - also please, can we update `cainome` to 0.4.0 and fix the errors?

File: src/services/monitoring.rs (hunk 1 above)
Comment (azurwastaken) [reply src/services/monitoring.rs:145]: after further investigation, the remaining computation are needed even in Full mode

File: src/types/position.rs [hunk 2]
```diff
@@ -264,64 +263,32 @@ impl Position {
         liquidate_contract: Felt,
         amount_to_liquidate: BigDecimal,
         minimum_collateral_to_retrieve: BigDecimal,
-        profit_estimated: BigDecimal,
     ) -> Result<Vec<Call>> {
         // TODO: remove those line when vesu contract allow partial liquidation
```
Comment (akhercha) [inline src/types/position.rs:267]: Remove the TODO comment

**Review by akhercha — APPROVED** [review]
seems gud

**Pull request merged**